scripts/signedbuildskadi.sh

```diff
@@ -1,274 +1,311 @@
+  sudo cp /opt/Skadi/scripts/cdqr /usr/local/bin/cdqr
+  sudo chmod +x /usr/local/bin/cdqr
+
+  # Create CyLR directory
+  sudo mkdir /opt/CyLR/
+  sudo chmod 777 /opt/CyLR
+
+  # Installs and Configures CDQR and CyLR
+  sudo -E bash /opt/Skadi/scripts/update.sh
+}
+
+timesketch_configs () {
+  # Write TS and Postgres creds to .env file
+  cd /opt/Skadi/Docker/
+  echo TIMESKETCH_USER=$TIMESKETCH_USER > ./.env
+  echo TIMESKETCH_PASSWORD=$TIMESKETCH_PASSWORD >> ./.env
+  echo POSTGRES_USER=$POSTGRES_USER >> ./.env
+  echo POSTGRES_PASSWORD=$psql_pw >> ./.env
+  echo HEAP_SIZE=1g >> ./.env
+
+  # Configure /etc/hosts file so the host can use same names for each service as the TimeSketch Dockers
+  echo 127.0.0.1       elasticsearch |sudo tee -a /etc/hosts
+  echo 127.0.0.1       postgres |sudo tee -a /etc/hosts
+  echo 127.0.0.1       neo4j |sudo tee -a /etc/hosts
+  echo 127.0.0.1       redis |sudo tee -a /etc/hosts
+
+  # Write TimeSketch config file on host
+  sudo cp /opt/Skadi/Docker/timesketch/timesketch.conf /etc/
+  sudo sed -i "s@SECRET_KEY = '<KEY_GOES_HERE>'@SECRET_KEY = '$SECRET_KEY'@g" /etc/timesketch.conf
+  sudo sed -i "s@<USERNAME>\:<PASSWORD>@$POSTGRES_USER\:$psql_pw@g" /etc/timesketch.conf
+  sudo sed -i "s@NEO4J_USERNAME = 'neo4j'@NEO4J_USERNAME = '$neo4juser'@g" /etc/timesketch.conf
+  sudo sed -i "s@NEO4J_PASSWORD = '<NEO4J_PASSWORD>'@NEO4J_PASSWORD = ''@g" /etc/timesketch.conf
+  sudo sed -i "s/UPLOAD_ENABLED = False/UPLOAD_ENABLED = True/g" /etc/timesketch.conf
+  sudo sed -i "s/GRAPH_BACKEND_ENABLED = False/GRAPH_BACKEND_ENABLED = True/g" /etc/timesketch.conf
+  sudo sed -i "s#@localhost/timesketch#@postgres/timesketch#g" /etc/timesketch.conf
+  sudo sed -i "s/ELASTIC_HOST = '127.0.0.1'/ELASTIC_HOST = 'elasticsearch'/g" /etc/timesketch.conf
+  sudo sed -i "s@'redis://127.0.0.1:6379'@'redis://redis:6379'@g" /etc/timesketch.conf
+  sudo sed -i "s/NEO4J_HOST = '127.0.0.1'/NEO4J_HOST = 'neo4j'/g" /etc/timesketch.conf
+
+  # Setup Nginx Auth
+  echo $NGINX_PASSWORD | sudo htpasswd -i -c /etc/nginx/.skadi_auth $NGINX_USER
+}
+
+clean_APT () {
+  # Clean APT
+  sudo apt-get -y autoremove --purge
+  sudo apt-get -y clean
+  sudo apt-get -y autoclean
+}
+
+containers_up () {
+  # Deploy the Skadi solution defined in ./docker-compose.yml
+  sudo docker-compose up -d
+
+  # Create a template in ES that sets the number of replicas for all indexes to 0
+  echo "Waiting for ElasticSearch service to respond to requests"
+  until $(curl --output /dev/null --silent --head --fail http://localhost:9200); do
+      printf '.'
+      sleep 5
+  done
+  echo "Setting the ElasticSearch default number of replicas to 0"
+
+  curl -XPUT 'localhost:9200/_template/number_of_replicas' \
+      -d '{"template": "*","settings": {"number_of_replicas": 0}}' \
+      -H'Content-Type: application/json'
+
+  echo "Waiting for Kibana service to respond to requests"
+  until $(curl --output /dev/null --silent --head --fail http://localhost:5601); do

```

Comment:
this should be modified to use the nginx proxy since Kibana port will be proxied. `http://localhost/kibana` or make this a docker exec command to run inside the Kibana docker instance `docker exec -it kibana .....`

Reply:
updated to http://localhost/kibana for now and if we feel it's more beneficial to run inside the kibana dokcer instance we can always update